sparse/impl/KokkosSparse_par_ilut_numeric_impl.hpp

```diff
@@ -71,8 +73,8 @@ struct IlutWrap {
                                   LU_row_map);
 
     const size_type lu_nnz_size = kh.get_spgemm_handle()->get_c_nnz();
-    Kokkos::resize(LU_entries, lu_nnz_size);
-    Kokkos::resize(LU_values, lu_nnz_size);
+    Kokkos::realloc(Kokkos::WithoutInitializing, LU_entries, lu_nnz_size);
```

Comment:
Just for my understanding. Why do you need to use ```realloc```? ```Kokkos::WithoutInitializing``` can also be used with ```resize```, can't it? Or was ```resize``` deprecated?